src/refactorings/simplify-ternary/simplify-ternary.ts

```diff
@@ -0,0 +1,109 @@
+import { Editor, Code, ErrorReason } from "../../editor/editor";
+import { Selection } from "../../editor/selection";
+import * as t from "../../ast";
+
+export { simplifyTernary, hasTernaryToConvert };
+
+async function simplifyTernary(
+  code: Code,
+  selection: Selection,
+  editor: Editor
+) {
+  const updatedCode = updateCode(t.parse(code), selection);
+
+  if (!updatedCode.hasCodeChanged) {
+    editor.showError(ErrorReason.DidNotFoundTernaryToSimplify);
+    return;
+  }
+
+  await editor.write(updatedCode.code);
+}
+
+function hasTernaryToConvert(ast: t.AST, selection: Selection): boolean {
```

Comment:
I'm nitpicking on this one, but I'd rename it `hasTernaryToSimplify` to keep the same semantics. Could you change that? 😃